go/cmd/gh-ost/main.go

```diff
@@ -149,6 +150,13 @@ func main() {
 	if migrationContext.SwitchToRowBinlogFormat && migrationContext.AssumeRBR {
 		log.Fatalf("--switch-to-rbr and --assume-rbr are mutually exclusive")
 	}
+	if migrationContext.TestOnReplicaSkipReplicaStop {
+		if !migrationContext.TestOnReplica {
+			log.Fatalf("--test-on-replica-skip-replica-stop requires --test-on-replica to be enabled")
```

Comment:
This will crash the migration at its very end -- after potentially hours of working. Instead, we can verify this in `main.go` -- at the very beginning of the migration.
Otherwise looking good!